test/e2e/common/node/kubelet.go

```diff
@@ -214,3 +214,48 @@ var _ = SIGDescribe("Kubelet", func() {
 		})
 	})
 })
+
+var _ = SIGDescribe("Kubelet-privileged-namespace", func() {
```

Comment:
looking at other SIGDescribe() i see text without spaces:

https://github.com/neolit123/kubernetes/blob/847b2e1085f9f642292da9f3f1a56ffb5366ab86/test/e2e_node/runtimeclass_test.go#L92

`("Kubelet privileged namespace")`
should i change it like so?

Reply:
updated https://github.com/kubernetes/kubernetes/compare/e19dde02f4f7c5ef112758e4e4d25b5c43efed32..bc83cea0eb45a051c98125466b48e99287558941